src/components/contextual/stake/StakePreview.vue

```diff
@@ -89,8 +89,10 @@ const shareBalanceToDisplay = ref(
 watch(
   () => props.action,
   () => {
-    stakeActions.value =
-      props.action === 'stake' ? [stakeAction] : [unstakeAction];
+    if (props.action === 'stake') stakeActions.value = [stakeAction];
+    if (props.action === 'unstake') stakeActions.value = [unstakeAction];
+    if (props.action === 'restake')
+      stakeActions.value = [unstakeAction, stakeAction];
```

Comment:
Could we show a custom tooltip on hover here? Feels strange to say user will lose eligibility in this case, since the gauge has been deactivated.
![image](https://user-images.githubusercontent.com/34865315/191789589-ad7405e2-f671-4ca6-8d17-a9b615d2f499.png)

Reply:
Thank you, updated 👍